src/inputs/ToggleButton.tsx

```diff
@@ -0,0 +1,82 @@
+import { ReactNode, useRef, useState } from "react";
+import { useFocusRing, useHover, usePress, useSwitch, VisuallyHidden } from "react-aria";
+import { useToggleState } from "react-stately";
+import { Icon, IconKey, maybeTooltip, resolveTooltip } from "src/components";
+import { Css } from "src/Css";
+import { isPromise, useTestIds } from "src/utils";
+
+export interface ToggleButtonProps {
+  /** Input label */
+  label: string;
+  selected?: boolean;
+  onChange: ((selected: boolean) => void) | ((active: boolean) => Promise<void>);
```

Comment:
**suggestion:**  Can we use the same variable name for the two callbacks? Right now we have `selected` or `active`. I think `selected` makes sense.